website/docs/cheatsheet.md

```diff
@@ -28,21 +28,26 @@ Built Sierra code of this package will be written to `target/dev` directory.
 `scarb clean` cleans `target` directory.
 :::
 
-### Building CASM
+### Building CASM and textual Sierra
```

Comment:
This is a cheatsheet, we don't have to showcase all features here, and this particular one is very minor. Please revert changes in this file.